notebooks/en/structured_generation.ipynb

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
<p>Would it be ok to remove all this blank space?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Yep, I did it by editing the notebook as JSON!